internal/witness/cmd/witness/internal/http/server_test.go

```diff
@@ -137,7 +140,7 @@ func mustCreatePK(pkPem string) crypto.PublicKey {
 }
 
 func createTestEnv(w *witness.Witness) (*httptest.Server, func()) {
-	r := mux.NewRouter()
+	r := mux.NewRouter().UseEncodedPath()
```

Comment:
Strictly, should probably separate these bits to do with the path escaping out into another PR? (this one L280, and L353 I think?)

Reply:
I'm not really sure why this would need to be a separate PR, since it's just what allows me to test the concept in this PR (adding logIDs) using "real" logIDs that contain slashes.  The overall switch to path escaping was already done in a previous PR so this is just following that, no?